rllib/core/learner/tf/tf_learner.py

```diff
@@ -96,12 +97,49 @@ def compute_gradients(
         grads = tape.gradient(loss[self.TOTAL_LOSS_KEY], self._params)
         return grads
 
+    @override(Learner)
+    def postprocess_gradients(
+        self,
+        gradients_dict: Mapping[str, Any],
+    ) -> Mapping[str, Any]:
+        """Applies grad clipping depending on the optimizer config."""
+
+        # Clip by value (each gradient individually).
+        grad_clip_by_value = self._optimizer_config.get("grad_clip_by_value", None)
+        if grad_clip_by_value is not None:
+            gradients_dict = {
+                k: tf.clip_by_value(
+                    v, -grad_clip_by_value, grad_clip_by_value
+                ) for k, v in gradients_dict.items()
+            }
+
+        # Clip by L2-norm (per gradient tensor).
+        grad_clip_by_norm = self._optimizer_config.get("grad_clip_by_norm", None)
```

Comment:
are we going to allow users to clip gradients by value and then by norm? that doesn't sound right to me, but that is the behavior that has been enabled here.